client/container.go

```diff
@@ -704,7 +704,7 @@ func (c *Client) ContainerAnnounceUsedSpace(ctx context.Context, announcements [
 	return nil
 }
 
-// SyncContainerWithNetwork requests network configuration using passed client
+// SyncContainerWithNetwork requests network configuration using passed [NetworkInfoExecutor]
 // and applies it to the container. Container MUST not be nil.
```

Comment:
can we take the best from both docs? `Pool` had `Pool and container MUST not be nil.` so that can be `Container and [NetworkInfoExecutor] MUST NOT be nil.` then

Reply:
I even suggest another solution. Why are we mentioning the container must not be empty if the function is for container updating? It is obvious. I removed the commend and added an error for this case.
Also updated documentation